### ISSUE TITLE
Bird Sounds are incorrect

### ISSUE THREAD

**Issue by ColinDavies71** [MiSTer-devel/Arcade-Phoenix_MiSTer#17]
The Bird sounds on levels 3 and 4 sound very odd and are two tone when on a real PCB the pitch kind of smoothly sweeps / glides up and down.  Hope this helps.

**Comment by Eebobobo** [comment]
> The Bird sounds on levels 3 and 4 sound very odd and are two tone when on a real PCB the pitch kind of smoothly sweeps / glides up and down. Hope this helps.

I noticed this too. Same with the audio on the mothership as it drops into the screen, with that sounding very unlike the arcade too.

**Comment by MilanPolle** [comment]
@alanswx

**Comment by darfpga** [comment]
I agree. Being the original author of that code I'm going to look into that issue. Just be patient... 

Thanks for noticing that issue.

Dar.

@MilanPolle, @ColinDavies71, @alanswx, @Eebobobo

**Comment by Eebobobo** [comment]
> I agree. Being the original author of that code I'm going to look into that issue. Just be patient... 
> 
> 
> 
> Thanks for noticing that issue.
> 
> 
> 
> Dar.
> 
> 
> 
> @MilanPolle, @ColinDavies71, @alanswx, @Eebobobo 

I'm sure it goes without saying, but any work is massively appreciated. 

I wonder if Phoenix is a bit of a pig to get right, as the sound on MAME has never been correct either.

**Comment by darfpga** [comment]
@Eebobobo @ColinDavies71 @MilanPolle 
> 
> I'm sure it goes without saying, but any work is massively appreciated.

Thank you

> I wonder if Phoenix is a bit of a pig to get right, as the sound on MAME has never been correct either.

My work is mostly based on schematics study. I own no arcade board. Sometime I have to cross check with Mame source code. For Phoenix sound my "analysis" leads to nearly the same conclusion as Mame for the principe of how sounds are generated. My VHDL certainly lack of accuracy because of working with fixed point integer instead of floating point. There is currently some trial to get better accuracy for analog part reproduction. I certainly give them some trial within a few months. 

One way to help may be that someone with arcade board could record sounds and post it somewhere. An another way could be if analog specialist study schematics and argue about Mame's reproduction principe. Third way could be to make a spice simulation of each part.

Dar.
